### PR TITLE
Replace placeholder graphs with nice graphs from Grafana

### PR DESCRIPTION
**Pull request by levex**
As in labs, this should be working!

### CODE REVIEW

**Comment by levex** [comment]
Oh wait - it doesn't, somebody needs to go and create the dashboards. Let me get that set up.

File: ansible/playbooks/create-meter.yaml [hunk 1]
```diff
@@ -16,6 +16,7 @@
         - apt-transport-https
         - ca-certificates
         - curl
+        - wget
```
Comment (fangyi-zhou) [inline ansible/playbooks/create-meter.yaml]: Umm...

File: client/meter/routes/transaction.js [hunk 2]
```diff
@@ -6,6 +6,16 @@ router.post('/buy', (req, res) => {
   const body = req.body;
   const contract = body.contract;
   const amount = body.amount;
+
```
Comment (fangyi-zhou) [inline client/meter/routes/transaction.js]: Please move it to `blockchain.js`

File: ansible/playbooks/create-meter.yaml (hunk 1 above)
Comment (levex) [reply ansible/playbooks/create-meter.yaml]: just because i like wget

**Comment by levex** [comment]
No it doesn't, if the service file is in the /usr/lib/systemd folder as it is, thanks to dpkg

**Comment by fangyi-zhou** [comment]
Good

**Comment by levex** [comment]
Please review & merge!

File: client/meter/routes/transaction.js [hunk 3]
```diff
@@ -15,6 +16,16 @@ router.post('/sell', (req, res) => {
...
+  fetch('http://localhost:8086/write?db=energy', {
+    method: 'POST',
+    headers: {
+		'Accept': 'application/json',
+		'Content-Type': 'application/json',
+	      },
+    body: 'energy_sold amount=' + amount,
+  })
+
```
Comment (fangyi-zhou) [inline client/meter/routes/transaction.js]: remove pls

File: client/meter/blockchain.js [hunk 4]
```diff
@@ -7,6 +7,17 @@ const bigNumber = require('bignumber.js');
...
 
+function send_energy_metric(name, amount) {
+  fetch('http://localhost:8086/write?db=energy', {
+    method: 'POST',
+    headers: {
+		'Accept': 'application/json',
+		'Content-Type': 'application/json',
+	      },
+    body: name + ' amount=' + amount,
+  });
```
Comment (fangyi-zhou) [inline client/meter/blockchain.js]: Promise ignored here. Intended?

File: ansible/playbooks/create-meter.yaml [hunk 5]
```diff
@@ -61,6 +64,31 @@
              && echo "  password = [\"/energy/client_pass\"]" >> /energy/parity/res/config.toml
     - name: install truffle
       shell: npm install -g truffle
+    - name: get the influxdb package
+      get_url:
+        url: https://dl.influxdata.com/influxdb/releases/influxdb_1.4.2_amd64.deb
+        dst: /energy/influxdb.deb
+    - name: install influxdb
+      apt: deb="/energy/influxdb.deb"
```
Comment (fangyi-zhou) [inline ansible/playbooks/create-meter.yaml]: Did you forget to start influx service?

**Comment by levex** [comment]
All fixed, @fangyi-zhou please review & green.

**Pull request merged**